app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -245,8 +256,16 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
     }
   }
 
-  String _getTriageUrl(int? number) {
-    return 'https://flutter-gold.skia.org/cl/github/$number';
+  String _getTriageUrl(RepositorySlug slug, int? number) {
```

Comment:
We should probably change this to be non nullable. It would be invalid if this got a null PR number.